book/modules/module2.tex

```diff
@@ -485,7 +485,7 @@
 	\]
 	This system is underdetermined (there are four variables and three equations). If $\mathcal P_1$ 
 	and $\mathcal P_2$ indeed intersect in a line, we know there must an infinite number
-	of solutions to this system. After row reducing, we see
+	of solutions to this system. After row reducing\footnote{See Appendix 1, System (14) and (15).}, we see
```

Comment:
This is a great addition! But we need to use `\ref{...}` to refer to the appendices and particular equations, otherwise the book will become inconsistent if things are moved around.

Can you add `\label{...}`s in the appropriate places and then reference them?

Reply:
I think my new commits resolve this issue (I used the `\label`s that were already in the Appendix). I'm not sure how to compile the LaTeX locally to check my work though, so I hope it's fine.

Reply:
Do you have `texlive` installed? If so, you should be able to run `latexmk -lualatex -latexoption="-interaction nonstopmode -halt-on-error -file-line-error" linearalgebra-book` from a command prompt.

Reply:
I figured it out: cloning a repo on Windows doesn't create symlinks automatically, so `linearalgebra-book` was being treated like a regular .tex file. I've compiled a PDF now, and everything looks good.